rcl/test/rcl/test_time.cpp

```diff
@@ -681,35 +630,30 @@ TEST(CLASSNAME(rcl_time, RMW_IMPLEMENTATION), rcl_clock_add_jump_callback) {
   rcl_jump_callback_t cb = reinterpret_cast<rcl_jump_callback_t>(0xBEEF);
   void * user_data = reinterpret_cast<void *>(0xCAFE);
 
-  EXPECT_EQ(RCL_RET_INVALID_ARGUMENT, rcl_clock_add_jump_callback(clock, threshold, NULL, NULL));
+  EXPECT_EQ(RCL_RET_INVALID_ARGUMENT, rcl_clock_add_jump_callback(&clock, threshold, NULL, NULL));
```

Comment:
To ensure that the invalid_argument refers to a missing jump_callback, can you set `user_data` appropriately here.

Reply:
Good point. See https://github.com/ros2/rcl/pull/685/commits/f72b7e87dea794f10c749363b68733632fb084fd.